### PR TITLE
Add best-effort support for COLRv1

### PR DESCRIPTION
**Pull request by LaurenzV**

### CODE REVIEW

**Comment by RazrFalcon** [comment]
Ready to merge?

File: crates/usvg/src/tree/text.rs [hunk 1]
```diff
@@ -565,7 +565,20 @@ impl Text {
         self.abs_stroke_bounding_box
     }
 
-    /// Text converted into paths, ready to render.
+    /// Text converted into paths, ready to render. Note that this is only a
```
Comment (RazrFalcon) [inline crates/usvg/src/tree/text.rs:568]: Can you format it a bit better? Check out `cargo docs` output. It would be formatted as a one long line.

Comment (LaurenzV) [reply crates/usvg/src/tree/text.rs:568]: Not sure I follow, this is how it looks for me, what exactly is wrong with that? Should I add a new line after the first sentence?

image

Comment (RazrFalcon) [reply crates/usvg/src/tree/text.rs:568]: >Should I add a new line after the first sentence?

Yes.

Comment (LaurenzV) [reply crates/usvg/src/tree/text.rs]: Done.

**Comment by LaurenzV** [comment]
> Ready to merge?

Yep.

**Comment by LaurenzV** [comment]
> I plan to make a resvg release soon. Do you have any more changes planned?

Well, ideally it would've been nice to figure out the font selection problem (#754) before the next release, but it it looks like it will still take some time and I think @laurmaedje is currently busy with other stuff right now, so I guess there is nothing holding us off from making a new release right now.

**Comment by RazrFalcon** [comment]
Ok, I will take a look at #754 myself this week and if it would not work will release resvg as is.

**Pull request merged**